certbot/display/util.py

```diff
@@ -39,12 +39,19 @@ def _wrap_lines(msg):
     :rtype: str
 
     """
+    wrapper = textwrap.TextWrapper(
```

Comment:
In case you're unaware, you can pass these values directly to `textwrap.fill` if you want instead of creating a `TextWrapper` object. Either approach is fine though.